stable/external-dns/templates/deployment.yaml

```diff
@@ -152,6 +152,11 @@ spec:
         - --rfc2136-insecure
           {{- end }}
         {{- end }}
+        # PowerDNS arguments
+        {{- if eq .Values.provider "pdns" }}
+        - --pdns-server={{ required "pdns.apiUrl must be supplied for provider 'pdns'" .Values.pdns.apiUrl }}:{{ .Values.pdns.apiPort }}
```

Comment:
Move this validation to the `_helpers.tpl` file. See how we implemented the macros `external-dns.validateValues` and `external-dns.validateValues.aws` for instance.